AdvancedPlaybackSampleApp/app/src/main/java/com/ooyala/sample/utils/SampleVideoPlayerFactory.java

```diff
@@ -0,0 +1,49 @@
+package com.ooyala.sample.utils;
+
+import com.ooyala.android.OoyalaException;
+import com.ooyala.android.item.Stream;
+import com.ooyala.android.player.MoviePlayer;
+import com.ooyala.android.player.PlayerFactory;
+import com.ooyala.android.player.StreamPlayer;
+
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Created by achaudhari on 9/9/16.
+ */
```

Comment:
please add a little documentation on each public class, to explain what it is for